lib/src/channel_preview.dart

```diff
@@ -128,7 +128,7 @@ class ChannelPreview extends StatelessWidget {
       stream: channel.state.messagesStream,
       initialData: channel.state.messages,
       builder: (context, snapshot) {
-        final lastMessage = channel.state.lastMessage;
+        final lastMessage = snapshot.data?.lastWhere((m) => m.shadowed != true);
         if (lastMessage == null) {
           return SizedBox();
```

Comment:
Should we fetch last non-shadow banned message instead? Doing this would make it impossible for normal users to see last message because of a person who is shadow banned.

Reply:
that's what we're doing, isn't it?